homeassistant/components/amberelectric/sensor.py

```diff
@@ -229,6 +259,16 @@ async def async_setup_entry(
             AmberPriceDescriptorSensor(coordinator, description, channel_type)
         )
 
+    for channel_type in actuals:
+        description = SensorEntityDescription(
+            key="actuals",
+            name=f"{entry.title} - {friendly_channel_type(channel_type)} Actual",
```

Comment:
I'm worried the naming will be confusing, and will result in people using this sensor instead of the `Price` sensor.

Can we change `Actual` to `Previous`